src/external/staticstruct.hh

```diff
@@ -26,6 +26,7 @@
 #include <limits>
 #include <memory>
 #include <stack>
+#include <cstdint>
```

Comment:
Can you help me from which library this is coming from to provide a fix upstream?

Reply:
std::uint32_t is used in line 131

Reply:
Thanks PR! staticstruct is now legacy and not used int tinyusdz. We should remote it. Let me give some time to work on removing it.

Reply:
FYI staticstruct git repo is here: https://github.com/syoyo/staticstruct